packages/veritone-widgets/src/widgets/SDOTable/story.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
```

Comment:
This should be refactored to use the BaseStory to abstract some of the boilerplate